runtime/src/main/cpp/Arrays.cpp

```diff
@@ -23,6 +23,12 @@
 #include "Natives.h"
 #include "Types.h"
 
+extern "C" {
+
+void checkRangeIndexes(KInt from, KInt to, KInt size);
```

Comment:
Just `extern "C" void checkRangeIndexes(KInt from, KInt to, KInt size);` would do.